### DISCUSSION TITLE
Adding additional information to "game loaded" popup.

### DISCUSSION THREAD

**Discussion by OlroxIX** [RetroAchievements/RAIntegration#1025] [Ideas]
Please add notification similar to this:
'Unsupported core. Achievements may not work. Tickets will be closed and manual unlock requests will be denied.' 
if game was loaded with unsupported core like Nestopia NES, Yabause Sega Saturn etc.

**Comment by Jamiras** [comment]
Firstly, this is not an RAIntegration issue as it doesn't affect RANes/RASnes/RAppleWin/etc. It could be an RAlibretro thing, but it also affects RetroArch, so it should either be in rcheevos (client-side) or RAWeb (server-side). It's actually most likely just a RetroArch thing as RAlibretro requires jumping through some additional hoops to even be able to see cores that aren't explicitly approved.

Secondly, we can't just blindly block cores without a good reason. A core update may fix whatever problem makes the core unsupported, and then we'd be popping up the message incorrectly.

Most unsupported cores are due to not exposing memory, which the runtime will detect. The associated achievements will be marked as unsupported, and the game placard will indicate a compatibility issue:

![image](https://github.com/RetroAchievements/RAIntegration/assets/32680403/422a27cb-4490-495e-a473-5624831c0104)

If users don't see that, they're not going to see any other popups we provide.
